.gitignore

```diff
@@ -1,3 +1,5 @@
 **/target
 Cargo.lock
 vendor
+.dep-*
+.protoc-*
```

Comment:
At least in my setup, these files are placed in the `target` directory, which is already git-ignored.

```
$ make all
target/dep-0.4.1 ensure
...
target/protoc-3.6.0 -I proto --go_out="plugins=grpc:/Users/kl/workspace/go/src" proto/destination.proto
...
```

Are you seeing something different in your setup?

Reply:
Ah, this must be cruft from when i was setting up the repo. disregard